lib/spack/spack/binary_distribution.py

```diff
@@ -743,8 +761,9 @@ def get_buildfile_manifest(spec):
                 continue
 
         elif relocate.needs_text_relocation(m_type, m_subtype):
-            data["text_to_relocate"].append(rel_path)
-            continue
+            if file_matches_any_binary_regex(abs_path, compiled_prefixes):
```

Comment:
one level less with `needs_text_relocation and file_matches_...` :laughing: sorry